### DISCUSSION TITLE
How do I set the keepAlive time?

### DISCUSSION THREAD

**Discussion by debraj-manna** [LogNet/grpc-spring-boot-starter#387] [Q&A]
Can someone let me know how do I set the `keepAliveTime` as mentioned in the official [grpc-java example](https://github.com/grpc/grpc-java/blob/45ffb06f00ffa782eb890202c6a7de1c8c809b43/examples/src/main/java/io/grpc/examples/keepalive/KeepAliveServer.java#L60)?

Is it by setting `grpc.netty-server.keep-alive-time` in `application.properties`? For example `grpc.netty-server.keep-alive-time=130s`

**Comment by jvmlet** [comment]
Yes, as described here https://github.com/LogNet/grpc-spring-boot-starter#3-usage
